examples/datasets/plot_iris_dataset.py

```diff
@@ -73,3 +84,5 @@
 ax.zaxis.set_ticklabels([])
 
 plt.show()
+
+# %%
```

Comment:
would be nice to add an explanation of the plot here to explain what the user sees.

Reply:
Thanks for the comments! I made some adjustments, hopefully I'll get a green build now :)